pom.xml

```diff
@@ -98,11 +98,21 @@
             <artifactId>spring-security-oauth2-jose</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-databind</artifactId>
-            <version>2.13.4.2</version>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>jackson-module-jaxb-annotations</artifactId>
```

Comment:
I've finally gotten around to attempt to implement the Spring 6 version of this library at UVA (thanks for getting this started!) but was wondering what the reason was for switching from jackson-databind to jackson-module-jaxb-annotations?  Jackson-module-jaxb-annotations seems to be bringing in some jackson classes that still use javax packages which causes conflicts with spring-boot-starter-data-jpa (Hibernate tries to use Jackson but uses the version of the class from jackson-module-jaxb-annotations).  However if I switch this back to jackson-databind then everything seems fine.  I can always put an exclusion in my pom's dependency but was curious what prompted this in the first place, thanks!

Reply:
I had the same experience with the jackson-module-jaxb-annotations artifact.  It feels "wrong" and caused issues.

Reply:
Thanks for pointing this out, fixed.